### PR TITLE
[gazebo] Resupport gazebo environment

### PR DESCRIPTION
**Pull request by mqcmd196**

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch_gazebo.noetic.repos [hunk 1]
```diff
@@ -0,0 +1,9 @@
+repositories:
+  ZebraDevs/fetch_gazebo:
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch_gazebo.noetic.repos:2]: humm, it seems fetch_robotics trasfer the repo to ZebraDevs, https://github.com/orgs/fetchrobotics/repositories, if there are conversation on this topic, please add that information.
if this repo is working on noetic, please add comments to https://github.com/ZebraDevs/fetch_gazebo/issues/112 and also ask for the release.

File: jsk_fetch_robot/jsk_fetch_gazebo.noetic.repos [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    type: git
+    url: git@github.com:ZebraDevs/fetch_gazebo
+    version: gazebo11
+  GT-RAIL/robot_pose_publisher:
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch_gazebo.noetic.repos:6]: which launch file uses this node? https://github.com/search?q=repo%3AZebraDevs%2Ffetch_gazebo%20robot_pose_publisher&type=code

File: jsk_fetch_robot/README.md [hunk 3]
```diff
@@ -357,7 +375,7 @@ c.f.
 furushchev@fetch15:/var/log/ros$ ls -lFahd
 drwxrwsr-x 381 ros ros 36K Nov  2 01:45 ./
 furushchev@pr2:/var/log/ros$ ls -lFahd
-drwxr-xr-x 6 ros ros 36K 11月  1 15:25 ./
+drwxr-xr-x 6 ros ros 36K 11 月  1 15:25 ./
```
[the remark this answers is not in the document]
Comment (mqcmd196) [reply jsk_fetch_robot/README.md:378]: thank you. removed

**Comment by mqcmd196** [comment]
@OldVincent 
Now, I'm creating a gazebo environment for Fetch at this PR. Please try if you have time. 

PR2 needs more patches...

**Comment by mqcmd196** [comment]
```shell
roslaunch jsk_fetch_startup fetch_gazebo_73b2.launch
```

**Comment by OldVincent** [comment]
Upstream library `fetch_gazebo` needs slight modification to run on Noetic. In file `fetch_gazebo/fetch_gazebo/launch/include/fetch.launch.xml`, line 12 `$(find xacro)/xacro.py` need to be replaced by `$(find xacro)/xacro`, then at least 'simulation.launch' can function normally; still 'playground.launch' gets stuck in loading because of waiting for responses from services: `waitForService: Service [/gazebo/set_physics_properties] has not been advertised, waiting...`

After install package `jsk_maps` from source, the room map and fetch robot can be loaded into gazebo correctly, yet the console is flooded with red errors. Most of these errors are from nodes 'tweet' and 'respeaker_node' in their failure-restarting loop.

Screenshot of partial error message from respeaker node:
Error of respeaker_node

Screenshot of partial error message from tweet node:
Error of tweet

**Comment by OldVincent** [comment]
> ```shell
> roslaunch jsk_fetch_startup fetch_gazebo_73b2.launch
> ```

Screenshot of running that launch file after minor fixes mentioned before:
Gazebo View

**Comment by mqcmd196** [comment]
I'm really really sorry, but I made wrong branch... Please check #1912

**Pull request closed without merging**